qml/keys/PressArea.qml

```diff
@@ -53,6 +53,8 @@ MultiPointTouchArea {
         holdTimer.stop();
     }
 
+    enabled: Keyboard.shown
```

Comment:
The `KeyboardGeometry` class is already exposed as a singleton, as `MaliitGeometry` so would it not be better to just use the `shown` property of that, rather than adding another that has to be proxied through the `InputMethod` object?

Reply:
Addressed, it would be useful not to change the name of classes on QML.